docs/tutorial/work-pools.md

```diff
@@ -12,49 +12,25 @@ search:
 
 # Work Pools
 
-## Why work pools?
+This part of the tutorial covers handling more complex and dynamic infrastructure requirements for your workflows using  [Prefect Cloud](https://www.prefect.io/cloud); alternatively, if you aren't using Prefect Cloud, see [Worker-based work pools](/tutorial/workers/). For simpler use cases, see [Deploying flows](/tutorial/deployments/).
 
-Work pools are a bridge between the Prefect orchestration layer and infrastructure for flow runs that can be dynamically provisioned.
-To transition from persistent infrastructure to dynamic infrastructure, use `flow.deploy` instead of `flow.serve`.
+## Key concepts
 
-!!! tip "[Choosing Between `flow.deploy()` and `flow.serve()`](/concepts/deployments/#two-approaches-to-deployments)"
-    Earlier in the tutorial you used `serve` to deploy your flows.
-    For many use cases, `serve` is sufficient to meet scheduling and orchestration needs.
-    Work pools are **optional**.
-    If infrastructure needs escalate, work pools can become a handy tool.
-    The best part?
-    You're not locked into one method.
-    You can seamlessly combine approaches as needed.
+* Work pool: An optional configuration that defines the infrastructure for executing flow runs. You can change these configurations as your infrastructure needs change.
+* Infrastructure block: Defines the specific infrastructure configuration that a work pool (such as a container image, Kubernetes manifest) uses.
+* Prefect-managed work pools are a great way to get started with Prefect and run workflows remotely. You don't need a cloud-provider account, such as AWS, with a Prefect-managed work pool.
```

Comment:
This is good content but I don't think that I would consider Prefect-managed work pools a key concept, but rather a type of work pool.